database/mongoSeed.js

```diff
@@ -0,0 +1,100 @@
+let db = require('./mongo.js');
+let mongoose = require('mongoose');
+const randomNumberUpTo = (limit) => Math.floor(Math.random() * limit);
+
+var listingSchema = new mongoose.Schema({
+  id: Number,
+  price: Number,
+  maxGuests: Number,
+  minStay: Number,
+  stars: mongoose.Decimal128,
+  numRatings: Number,
+  bookedDates: [String]
+})
+
+var Listing = mongoose.model('Listing', listingSchema);
+
+// create listings and insert in batches of 100
+const seedListings = async () => {
+  let t1 = Date.now();
+  for (let batch = 0; batch < 100000; batch++) {
+    let listingBatch = [];
+    for (let listing = 0; listing < 100; listing++) {
+      let id = (batch * 100) + (listing + 1);
+      let price = 50 + randomNumberUpTo(400);
+      let maxGuests = 1 + randomNumberUpTo(6);
+      let minStay = 1 + randomNumberUpTo(2);
+      let stars = (1 + (Math.random() * 4)).toFixed(2);
```

Comment:
I believe that `.toFixed()` will be turning this into a string. You should double-check, and if so you can convert back to a number to be consistent with your other variables.